### DISCUSSION TITLE
Change log-viewer location

### DISCUSSION THREAD

**Discussion by mlezajic** [sevdokimov/log-viewer#123] [Q&A] [answered]
Hi Sergey,

is it somehow possible to change the location of the `.log-viewer` directory? The default appears to be the user's home directory.

Many thanks!

**Comment by sevdokimov** [comment, marked as answer]
You can specify `log-viewer.config-dir=/log-viewer/cache`  property in `config.conf`. 

(the name of the property looks weird, I'll rename it to `log-viewer.cache-dir` later)

**Reply by mlezajic** [reply]
Thanks Sergey for the quick response! I am using spring boot, so may I configure this in the `application.properties` as well?

**Reply by mlezajic** [reply]
@sevdokimov setting this in `application.properties` works! Many thanks for your help.
